Sources/FoundationNetworking/URLSession/URLSessionDelegate.swift

```diff
@@ -128,6 +128,8 @@ public protocol URLSessionTaskDelegate : URLSessionDelegate {
      func urlSession(_ session: URLSession, task: URLSessionTask, didCompleteWithError error: Error?)
     
      func urlSession(_ session: URLSession, task: URLSessionTask, willBeginDelayedRequest request: URLRequest, completionHandler: @escaping (URLSession.DelayedRequestDisposition, URLRequest?) -> Void)
+    
+    func urlSession(_ session: URLSession, task: URLSessionTask, didFinishCollecting metrics: URLSessionTaskMetrics)
```

Comment:
nit: formatting seems to be inconsistent here

Reply:
The other formatting is inconsistent: the first two methods are indented 4 spaces, the others 5. 4 seems correct.

Reply:
Sure, the nit is about the rest of the code being inconsistent, not yours 😁

Good opportunity to make the rest of the functions use 4-spaces indentation?

Reply:
Sure. Fixed all of the spacing issues in this file in https://github.com/apple/swift-corelibs-foundation/pull/2886/commits/75bcd79a15e4234f3d4437c3338ab721b9a8f792.

Reply:
Thanks!